### PR TITLE
Add bower dependencies

### LINKED ISSUE #134: Undefined error when using .get()
It seems the ErrorStackParser variable is undefined, throwing an error when required instead of generating a complete stacktrace when using the get method. 

I'm not sure if the issue is my fault or not, some guidance would be appreciated.

Here's the..er..stacktrace of the error itself.

```
TypeError: Cannot read property 'parse' of undefined at Object.<anonymous> (http://localhost:3000/libs/stacktrace-js/stacktrace.js:79:51) 
at Object.StackTrace$$fromError [as fromError] (http://localhost:3000/libs/stacktrace-js/stacktrace.js:78:20) 
at Object.StackTrace$$get [as get] (http://localhost:3000/libs/stacktrace-js/stacktrace.js:63:33) 
at Object.processErrorCode 
...
```

And here's my code block that the issue originates from.

```
  StackTrace.get()
                    .then(function (stackframes) {
                        var stringifiedStack = stackframes.map(function (sf) {
                            return sf.toString();
                        }).join('\n');
                        Logger.getErrors().addMessage(errorMessage, stringifiedStack, "ErrorCode");
                    })
                    .catch(function (err) {
                        Logger.getErrors().addMessage(errorMessage, err.stack, "ErrorCode");
                    });
```

### PR DESCRIPTION
**Pull request by oliversalzburg**
Fixes #134

### CODE REVIEW

**Comment by eriwen** [comment]
:+1:

**Comment by oliversalzburg** [comment]
@eriwen We should probably tag stacktrace-gps and stacktrace.js with a patch update and update the bower.json to that version. Then the dependencies should be pulled in.

Another option I was considering was to specify the `.min.js` version as the `main` file. That one contains all the dependencies as well.

**Comment by eriwen** [comment]
@oliversalzburg Yes, we should tag and publish a new release for those projects. You no longer have to wait for me to publish to npm! Make sure to bump and push gps, then bump the versions in stacktrace.js, tag and push that.

Regarding the bower dependencies, we used `.min.js` for component.json, so it makes sense to do the same here. Thanks!

**Comment by oliversalzburg** [comment]
@eriwen Okay, so, if we set the `.min.js` versions in the `main` section, then we don't really need to specify other packages as dependencies. We should set the referenced packages as `devDependencies` then instead.

Unless you disagree, I would make that change, merge, tag, push and npm publish :)

File: bower.json [hunk 1]
```diff
@@ -27,5 +27,10 @@
         "**/.*",
         "node_modules",
         "spec"
-    ]
+    ],
+    "dependencies": {
+        "error-stack-parser": "~1.2.3",
+        "stack-generator": "~1.0.7",
+        "stacktrace-gps": "~2.2.1"
```
Comment (eriwen) [inline bower.json]: Let's bump this to 2.3.0

**Comment by eriwen** [comment]
Once we can rebase this PR on the HEAD of master and update the stacktrace-gps dependency, you can merge this and push a new version @oliversalzburg. Thanks!

**Comment by oliversalzburg** [comment]
I didn't change the `main` to the minified build as the `component.json` actually doesn't reference it either. Not sure how to proceed here. I still think it would make sense to make that change, but I wanted to delay it until a working bower setup is in place.

I pushed a 1.0.1 tag and published to npm. I didn't touch the dependencies as a `bower i stacktrace-js` seems to already perform as intended. Not sure if anything else is actually needed.

I'll probably reference the bower dependency in our own projects very soon and then I'll get some real-world experience and adjust accordingly, if necessary.

```
$ bower i stacktrace-js
bower stacktrace-js#*           cached git://github.com/stacktracejs/stacktrace.js.git#1.0.0
bower stacktrace-js#*         validate 1.0.0 against git://github.com/stacktracejs/stacktrace.js.git#*
bower stacktrace-js#*              new version for git://github.com/stacktracejs/stacktrace.js.git#*
bower stacktrace-js#*          resolve git://github.com/stacktracejs/stacktrace.js.git#*
bower stacktrace-js#*         download https://github.com/stacktracejs/stacktrace.js/archive/1.0.1.tar.gz
bower stacktrace-js#*          extract archive.tar.gz
bower stacktrace-js#*         resolved git://github.com/stacktracejs/stacktrace.js.git#1.0.1
bower stacktrace-gps#~2.3.0 not-cached git://github.com/stacktracejs/stacktrace-gps.git#~2.3.0
bower stacktrace-gps#~2.3.0    resolve git://github.com/stacktracejs/stacktrace-gps.git#~2.3.0
bower stack-generator#~1.0.7    cached git://github.com/stacktracejs/stack-generator.git#1.0.7
bower stack-generator#~1.0.7  validate 1.0.7 against git://github.com/stacktracejs/stack-generator.git#~1.0.7
bower error-stack-parser#~1.2.3 cached git://github.com/stacktracejs/error-stack-parser.git#1.2.3
bower error-stack-parser#~1.2.3         validate 1.2.3 against git://github.com/stacktracejs/error-stack-parser.git#~1.2.3
bower stacktrace-gps#~2.3.0             download https://github.com/stacktracejs/stacktrace-gps/archive/2.3.0.tar.gz
bower stackframe#~0                       cached git://github.com/stacktracejs/stackframe.git#0.3.1
bower stackframe#~0                     validate 0.3.1 against git://github.com/stacktracejs/stackframe.git#~0
bower stacktrace-gps#~2.3.0              extract archive.tar.gz
bower stacktrace-gps#~2.3.0             resolved git://github.com/stacktracejs/stacktrace-gps.git#2.3.0
bower source-map#0.5.1                    cached git://github.com/mozilla/source-map.git#0.5.1
bower source-map#0.5.1                  validate 0.5.1 against git://github.com/mozilla/source-map.git#0.5.1
bower stacktrace-js#~1.0.1               install stacktrace-js#1.0.1
bower error-stack-parser#~1.2.3          install error-stack-parser#1.2.3
bower stack-generator#~1.0.7             install stack-generator#1.0.7
bower stackframe#~0                      install stackframe#0.3.1
bower stacktrace-gps#~2.3.0              install stacktrace-gps#2.3.0
bower source-map#0.5.1                   install source-map#0.5.1
```

**Comment by eriwen** [comment]
@oliversalzburg Yeah, this works just fine. The only possible hangup I can see is the complexity of including all dependencies if a consumer does not use the minified version at `bower_components/stacktrace-js/dist/stacktrace.min.js`

**Pull request merged**